### ISSUE TITLE
Visibility panel does not reflect current settings upon edit

### ISSUE THREAD

**Issue by vantuyls** [osulp/Scholars-Archive#1501]
When editing the Visibility panel, it does not display all of the information related to visibility for the item. 

To Replicate:
- Create a work with an embargo
- save work
- edit work
- see that visibility panel does not include some information

<img width="473" alt="Screen Shot 2019-04-10 at 1 46 21 PM" src="https://user-images.githubusercontent.com/9534576/55912206-16aaea80-5b97-11e9-9eb4-fe6117ab5e6f.png">

**Comment by simholt** [comment]
Hmmmm, it looks like the embargo reason is still not sticking.

I deposited an item, selected a reason, and saved it but the reason did not show up. 
![Screen Shot 2019-09-10 at 2 40 03 PM](https://user-images.githubusercontent.com/33705514/64652784-8a1c8680-d3d9-11e9-9cab-b8ae98e3e40c.png)



I clicked edit and added the reason again and it appeared.
![Screen Shot 2019-09-10 at 2 40 47 PM](https://user-images.githubusercontent.com/33705514/64652796-91439480-d3d9-11e9-924e-7c19f6527b86.png)



https://ir-staging.library.oregonstate.edu/concern/undergraduate_thesis_or_projects/qv33rw91s
